oracle-explorer-client/src/main/scala/org/bitcoins/explorer/model/CreateAttestations.scala

```diff
@@ -0,0 +1,13 @@
+package org.bitcoins.explorer.model
+
+import org.bitcoins.core.protocol.tlv.OracleAttestmentV0TLV
+import org.bitcoins.crypto.Sha256Digest
+
+case class CreateAttestations(
+    announcementHash: Sha256Digest,
+    attestment: OracleAttestmentV0TLV) {
+
+  override def toString: String = {
+    s"attestations=${attestment.hex}"
```

Comment:
Does this need the `announcementHash`?

Reply:
No, it's part the URI that needs to be built though. The endpoint is `/v1/events/:hash/attestation`